.changeset/grumpy-crabs-hide.md

```diff
@@ -0,0 +1,11 @@
+---
+"fluid-framework": minor
+"@fluidframework/tree": minor
+---
+---
+"section": tree
+---
+
+Add `SchemaFactoryAlpha.scopedFactory`
+
+Add `SchemaFactoryAlpha.scopedFactory` method as an easy way to create a new `SchemaFactory` with a nested scope string.
```

Comment:
Can you add a bit about the usage of this method since it's new?

Reply:
It has API documentation which details its use. I'd prefer to avoid duplicating that here, but we also can't link it from here until after this is merged and the docs are published. For now I've added a link to SchemaFactoryAlpha where the eventually docs in question will be linked from. I can then try and update that link once the docs are updated (since I believe they get updated before we do releases from main?)